Makefile

```diff
@@ -3,6 +3,8 @@ FMT_LOG=fmt.log
 
 GOPATH ?= "$(HOME)/go"
 
+UNIT_TEST_PACKAGES := $(shell go list ./pkg/...)
```

Comment:
Not very intuitive, but `go list` will make network connections and might end up touching go.mod/go.sum. Best would be to use `./pkg/... ` directly in the `go test` command instead. Some other tools might not support this notation, like some linters, which is why we resort to `go list` there.